src/test/scala/org/broadinstitute/dsde/firecloud/mock/MockWorkspaceServer.scala

```diff
@@ -158,6 +159,19 @@ object MockWorkspaceServer {
           .withBody(MockUtils.rawlsErrorReport(NotFound).toJson.compactPrint)
       )
 
+    MockWorkspaceServer.workspaceServer
+      .when(
+        request()
+          .withMethod("GET")
+          .withPath(s"${workspaceBasePath}/tags/autocomplete/abc")
```

Comment:
this path should now just be /tags. Actually ... if you're using the `stubRawlsService` util in the test, I don't think this is even needed.